### ISSUE TITLE
Add me as maintainer

### ISSUE THREAD

**Issue by chrisjsewell** [conda-forge/jupyter-cache-feedstock#2]
Hey @anirrudh I thought I didn't add this recipe yet lol.
Can you add me as a maintainer and also merge #1 thanks 😄

**Comment by chrisjsewell** [comment]
But update the nbclient pinning first!

**Comment by anirrudh** [comment]
@chrisjsewell Yes! Let's add you as a maintainer too - would you mind making a PR adding yourself?
